chmgr-lib/src/manager.rs

```diff
@@ -172,7 +172,34 @@ impl ManagerHandle {
         afb_log_msg!(Notice, self.event, "Valid idp-auth");
         Ok(())
     }
+    
 
+    pub fn set_payment_option(&self, msg: &Iso15118StateSelectedPaymentOption) -> Result<(), AfbError> {
+
+        let mut data_set = self.get_state()?;
+        
+        data_set.iso = match msg {
+            Iso15118StateSelectedPaymentOption::EIM => IsoState::Eim,
+            Iso15118StateSelectedPaymentOption::PNC => IsoState::Pnc,
+        };
+
+        Ok(())
+    }
+
+    fn charging_protocol(&self, data_set: &MutexGuard<ChargingState>) -> Result<(), AfbError>{
```

Comment:
Unless I am missing something, this function does nothing

Reply:
I didn't PR updates on display binding yet. So at the moment, the info that the function pushes is not used.

Reply:
Indeed, I read that part too quickly. :+1: